tests/kernel/semaphore/semaphore/src/main.c

```diff
@@ -140,13 +143,70 @@ void sem_take_multiple_high_prio_helper(void *p1, void *p2, void *p3)
 	k_sem_give(&high_prio_sem);
 }
 
+/* First function for mutual exclusion test */
+void sem_queue_mutual_exclusion1(void *p1, void *p2, void *p3)
+{
+	for (int i = 0; i < 5; i++) {
```

Comment:
Why use i and not the critical_var here?

Reply:
I'm using "for" loop to check that threads work as expected 5 times. Really, it will work and without that loop, but my idea was to test mutual exclusion multiple times (5 times), that why I'm using loop and that loop has "i" variable as a counter.